### ISSUE TITLE
Question: Status of Pretrained Model

### ISSUE THREAD

**Issue by asalbright** [Improbable-AI/walk-these-ways#22]
Hi there and thanks for providing the repo in addition to a Pretrained model. 

I am wondering where the Pretrained model comes from? What iteration, more or less? What is the status of the curriculum? 

When I inspect the status of the provided curriculum: 

[runs/gait-conditioned-agility/pretrain-v0/train/025417.456545/curriculum/distribution.pkl](https://github.com/Improbable-AI/walk-these-ways/blob/38bb76726ae3c3c8fe54aad72ffee2dd407787b7/runs/gait-conditioned-agility/pretrain-v0/train/025417.456545/curriculum/distribution.pkl)

I see that distribution["iteration"] returns 0, and looking at the distribution["distribution"], it seems the distribution is limited to values that are the initial values. [-1, 1] for the linear/angular velocities. 

I am curious because I would like to do some fine-tuning on your set of weights that seem to be working well enough. However, if I want to pick up training with those weights, I would assume I also want to pick up with the curriculum no?

**Comment by asalbright** [comment]
Realizing that the lack of a "current" or "matching" curriculum, doesn't seem to matter all that much, since the policy learns rather quickly to move up the curriculum since it can achieve all the reward thresholds. 

Still, I am interested as to where the policy came from.
